test_run_postgres.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+source ./.env
+
+docker run --name postgres-oauth2client \
```

Comment:
Could docker-compose be used instead of a shell script?

Reply:
I see your point. Changing to docker compose and mimicking **LTI** repo structure

Reply:
@fredmajor I would prefer having this on README rather than on shell script, but I think devops would argue about that, so docker-compose is a good compromise